ydb/core/kqp/common/kqp_resolve.cpp

```diff
@@ -7,14 +7,20 @@
 #include <ydb/core/tx/datashard/range_ops.h>
 #endif
 
+#include <yql/essentials/minikql/mkql_node_builder.h>
+
 namespace NKikimr {
 namespace NKqp {
 
 using namespace NMiniKQL;
 using namespace NYql;
 using namespace NYql::NNodes;
 
-NUdf::TUnboxedValue MakeDefaultValueByType(const NKikimr::NMiniKQL::TType* type) {
+NUdf::TUnboxedValue MakeDefaultValueByType(NKikimr::NMiniKQL::TType* type) {
+    bool isOptional;
```

Comment:
@dahbka-lis PTAL
I have fixed the ydb/core/kqp/ut/batch_operations tests with MSan enabled. You might need to add more types here.